integrated_channels/degreed2/client.py

```diff
@@ -129,11 +130,19 @@ def create_course_completion(self, user_id, payload):
             f'Attempting find course via url: {self.get_completions_url()}'),
             user_id
         )
-        return self._post(
+        code, body = self._post(
             self.get_completions_url(),
             json_payload,
             self.ALL_DESIRED_SCOPES
         )
+        if code == HTTPStatus.BAD_REQUEST.value:
+            error_response = json.loads(body)
+            for error in error_response['errors']:
+                if 'detail' in error and 'Invalid user identifier' in error['detail']:
+                    raise ClientError(f'Degreed2 create_course_completion failed due to '
```

Comment:
Are we  handling/logging this exception somewhere? Otherwise it would be best to just log it here instead of raising an exception.

Reply:
Yes - this exception will be catched here:
https://github.com/openedx/edx-enterprise/blob/3c1cda636e56979b0285b68a3da4f28c590c804f/integrated_channels/integrated_channel/transmitters/learner_data.py#L378

and `process_transmission_error` will log it.